src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -166,6 +166,7 @@ class AuthScreens extends React.Component {
             animationEnabled: true,
             gestureDirection: 'horizontal',
             cardOverlayEnabled: true,
+            presentation: 'transparentModal',
```

Comment:
Why do we need this? Can we add some comment maybe?

Reply:
This option is required to make previous screen visible underneath the modal screen
https://reactnavigation.org/docs/6.x/stack-navigator#transparent-modals

Adding a comment to the code now 👍